tfhe/c_api_tests/test_compressed_list.c

```diff
@@ -0,0 +1,145 @@
+#include "tfhe.h"
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+int main(void) {
+  int ok = 0;
+
+  // First, we create a ClientKey and a CompactPublicKey
+  ClientKey *client_key = NULL;
+  ServerKey *server_key = NULL;
+
+  {
+    ConfigBuilder *builder;
+    Config *config;
+    ok = config_builder_default(&builder);
+    assert(ok == 0);
+
+    config_builder_enable_compression(&builder, &COMP_PARAMS_FOR_MESSAGE_2_CARRY_2_KS_PBS);
```

Comment:
as a reminder use the custom parameters that correspond to the compression ones,

here there is a missing check on the OK value after then config_builder_enable_compression call

Reply:
Done